src/utils/filters/video-filter.js

```diff
@@ -0,0 +1,17 @@
+/** @format */
+
+export const videoFilter = (videos, filter = null) => {
+  if (filter === "allvideos") {
+    return videos;
+  } else if (filter.length > 0) {
+    return videos.filter(
+      (video) => video.category.toUpperCase() === filter.toUpperCase()
+    );
+  }
+  return videos;
+};
+// filter.length > 1
+//   ? videos.filter(
+//       (video) => video.category.toUpperCase() === filter.toUpperCase()
+//     )
+//   : videos;
```

Comment:
You can remove these comments if not going to use them.

Reply:
will remove it.